### ISSUE TITLE
Can't upload subtitles via API

### ISSUE THREAD

**Issue by bartoszsemanycz** [vimeo/vimeo.php#310]
Using Vimeo library version 3.0.8. Uploading subtitles stopped working (nothing was changed by our side). 

Function uploadTexttrack returns VimeoUploadException with response Error 500. Subtitle's name is created in repo, but subtitles doesn't exit and gives 404 when trying to download them.

Can anybody confirm if subtitles upload via API is working at the moment? A week ago it was all ok. Also checked on another server (clean installation). Traffic isn't blocked as video uploads, thumbnail uploads etc. working correctly. 

Response: 
```
Array
(
    [url] => https://captions.cloud.vimeo.com/captions/14547745?expires=1644846545&sig=c86fbc1db4b206ae51b43b30af48410d7fc86206
    [content_type] => text/html; charset=UTF-8
    [http_code] => 500
    [header_size] => 509
    [request_size] => 194
    [filetime] => -1
    [ssl_verify_result] => 0
    [redirect_count] => 0
    [total_time] => 0,144669
    [namelookup_time] => 0,028284
    [connect_time] => 0,034001
    [pretransfer_time] => 0,051108
    [size_upload] => 0
    [size_download] => 93
    [speed_download] => 642
    [speed_upload] => 0
    [download_content_length] => 93
    [upload_content_length] => -1
    [starttransfer_time] => 0,144621
    [redirect_time] => 0
    [redirect_url] => 
    [primary_ip] => 34.96.100.118
    [certinfo] => Array
        (
        )

    [primary_port] => 443
    [local_ip] => 5.135.25.137
    [local_port] => 53866
)

```

**Comment by UTSA123432** [comment]
We are having a similar issue where the VTT upload has stop working all of the sudden. Debugging the SDK also gives similar results to @bartoszsemanycz issue. Can you please assist?

**Comment by matthew-8925** [comment]
Just adding a +1, we are seeing the same issue - we're always getting a 500 error when trying to upload VTT subtitles via the API.
It was working a week ago.

**Comment by bartoszsemanycz** [comment]
Solved: https://github.com/vimeo/vimeo.php/pull/311

**Comment by aaronm67** [comment]
Closed by #311 in [v3.0.10](https://github.com/vimeo/vimeo.php/releases/tag/3.0.10)
